documentation/configuration/parameters/daprSecrets.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: documentation
+menu: configuration
+pill: daprSecrets
+subtitle: flyway.daprSecrets
+---
+
+# Dapr Secrets
+{% include teams.html %}
+
+## Description
+A comma-separated list of paths to key-value secrets in a
+[Dapr](https://docs.dapr.io/developing-applications/building-blocks/secrets/secrets-overview/) Secret Store that contain 
+Flyway configurations. 
+
+If multiple secrets specify the same configuration parameter, then the last secret takes precedence.
+
+Example: `secret1,secret2`
```

Comment:
We should probably remove this example, since it's not correct in every case and the usage section below covers it

Reply:
It ought to be the case that the last secret takes precedence? Have I missed something there?

Reply:
I meant it's incorrect in that it's not a comma-separated list always, such as in the API usage or Gradle usage